src/SingleCellAnnotator.imjoy.html

```diff
@@ -321,17 +328,32 @@
     },
     async uploadAnnotation(){
       // Default options are marked with *
+      const sendData = JSON.parse(JSON.stringify(sendDataTemplate));
+      sendData.discard = this.current_image_data.discarded;
```

Comment:
1. it should be `discarded`
2. According to the README, if discarded is true, then you can skip sending the rest fields.